io/net/bonding.py

```diff
@@ -112,6 +112,11 @@ def setUp(self):
         self.peer_wait_time = self.params.get("peer_wait_time", default=5)
         self.sleep_time = int(self.params.get("sleep_time", default=5))
         self.mtu = self.params.get("mtu", default=1500)
+        for root, dirct, files in os.walk("/root/.ssh"):
+            for file in files:
+                if file.startswith("avocado-master-root"):
```

Comment:
when user changes, the file name changes too.. example `avocado-master-hscroot` ... make it generic something like `avocado-master`

Reply:
same for padmin

Reply:
`file` is a builtin in python,  but it works fine on python3.